app/controllers/user/recordings_controller.rb

```diff
@@ -30,7 +30,7 @@ def create
 
   def destroy
     recording = current_user.recordings.find_by(id: params[:id])
-    if recording
+    if recording && recording.tours.empty?
```

Comment:
I don't think this is right. We should be able to delete a recording that is included on a tour. May need to add `dependent: :destroy` somewhere.

Reply:
Scenario: You pick someone's recording for a landmark in your tour and they delete that recording.
Issue: That landmark in your tour now doesn't have a recording. 

If you are going to let content creators use other people's recordings, we need a way to regulate those recordings so this issue doesn't happen.